master/docs/developer/config.rst

```diff
@@ -136,6 +136,11 @@ described in :ref:`developer-Reconfiguration`.
         The metrics configuration from :bb:cfg:`metrics`, or an empty
```

Comment:
on the graph, there could be as many graphical visualization than storage (even more)